src/common/centile.cc

```diff
@@ -0,0 +1,106 @@
+/*
+ * centile.cc
+ *
+ *  Created on: 25-Aug-2016
+ *      Author: prashant.kr
+ */
+#include "centile.h"
+
+namespace centile {
+  Centile::Centile(unsigned int start, unsigned int end, unsigned int inc) {
```

Comment:
put some comments around this class. What do the parameters mean? either check for argument sanity in the constructor ( example, can inc be greater than end - start ?) or document the assumptions on the input if you 'trust' the users of your class.